### PR TITLE
Calling method from another class.

### PR DESCRIPTION
**Pull request by SmartCode18**
Used constructor to call the method from other class.

### CODE REVIEW

File: phaseone/.idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fairwinds0099) [inline phaseone/.idea/misc.xml]: please commit only the files that you have made changes... 
I dont think misc.xml is a file you have created

File: phaseone/src/main/java/Utils.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+public class Utils {
+
+    public Utils () {
```
Comment (fairwinds0099) [inline phaseone/src/main/java/Utils.java]: You have a Utils class... In Utils class there are going to be methods like openSession(), connectToDatabase(), forLoopTenTimes() etc.

please remove Utils() method

File: phaseone/src/main/java/Utils.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+        forLoopTenTimes();
+    }
+
+    private void forLoopTenTimes() {
```
Comment (fairwinds0099) [inline phaseone/src/main/java/Utils.java]: why private?

File: phaseone/.idea/misc.xml (hunk 1 above)
Comment (SmartCode18) [reply phaseone/.idea/misc.xml]: sorry for the confusion...

**Comment by fairwinds0099** [comment]
Please pull master, open a new branch (fix_0), fix the problem, make it run
five times and make a PR with the fixed branch.

Do not merge :)

Bye

**Comment by SmartCode18** [comment]
Opened a branch but not able to fix the problem. Can you show me how to do it?

**Pull request merged**